renovate.json

```diff
@@ -61,6 +61,11 @@
         "github.com/keptn/lifecycle-toolkit/keptn-cert-manager"
       ],
       "groupName": "keptn/cert-manager"
+    },
+    {
+      "matchPackageNames": ["squidfunk/mkdocs-material"],
```

Comment:
My apologies, I was confused because both packages' names redirect to the same repository. So the correct way is 
`{
      "matchPackageNames": ["squidfunk/mkdocs-material",
"mkdocs-material"],
      "groupName": "squidfunk/mkdocs-material",
    }`?